lesson1/task_1.rb

```diff
@@ -0,0 +1,28 @@
+class Human
+
+  def initialize(firstname, lastname)
+    @firstname = firstname
+    @lastname = lastname
+  end
+
+  def get_name
+    return @firstname
```

Comment:
Using "return" is unnecessary if it is the last line to be executed in the method, since Ruby automatically returns the last evaluated expression